### PR TITLE
Server description and greeting changes

### PR DESCRIPTION
**Pull request by delebota**
Added server description textbox to the server start menu (#3288)
Added server greeting textbox to the server start menu

Fixed server greeting (#4913)

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 0.0.7 (in development)
 ------------------------------------------------------------------------
-- Improved: [#5137]: Removing all guests no longer closes the rides and removes the vehicles.
+- Improved: [#5137] Removing all guests no longer closes the rides and removes the vehicles.
+- Improved: [#3288] Added server description and greeting textboxes to the start server menu.
 - Fix: [#5150] --openrct-data-path sets user data path instead of OpenRCT2 data path.
+- Fix: [#4913] Server greeting displaying local setting.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: Can you order the changelog items by issue number like the previous releases?

Comment (delebota) [inline distribution/changelog.txt]: Sorry, didn't notice that. Will do!

File: src/openrct2/windows/server_start.c [hunk 2]
```diff
@@ -178,6 +191,12 @@ static void window_server_start_mouseup(rct_window *w, sint32 widgetIndex)
 	case WIDX_NAME_INPUT:
 		window_start_textbox(w, widgetIndex, STR_STRING, _name, 64);
 		break;
+	case WIDX_DESCRIPTION_INPUT:
+		window_start_textbox(w, widgetIndex, STR_STRING, _description, 256);
```
Comment (marijnvdwerf) [inline src/openrct2/windows/server_start.c]: Can you introduce a define for this number?

Comment (delebota) [inline src/openrct2/windows/server_start.c]: Sure!

**Comment by delebota** [comment]
Commits have been amended with the above suggestions.

File: contributors.md [hunk 3]
```diff
@@ -55,6 +55,7 @@ Includes all git commit authors. Aliases are GitHub user names.
 * Niels NTG Poldervaart (Niels-NTG) - Misc.
 * (zaxcav) - Improvements to original pathfinding algorithm.
 * Hielke Morsink (Broxzier) - Tile inspector, misc.
+* Thomas Delebo (delebota) - Multiplayer, UI, misc.
```
Comment (Nubbie) [inline contributors.md]: `Multiplayer` makes it sound like you did the whole online experience :p

Comment (delebota) [inline contributors.md]: Oh yeah, I guess it's a bit narrow I will just replace it all with misc.

**Comment by Gymnasiast** [comment]
@IntelOrca I rebased this, so it can be merged if it's up to your standards.

**Pull request merged**